plugin/src/cli_utils/localpvhostpath/mod.rs

```diff
@@ -0,0 +1,77 @@
+use clap::Parser;
+use plugin::ExecuteOperation;
+use crate::cli_utils::CliArgs;
+
+/// LocalPV Hostpath operations.
+#[derive(Parser, Debug)]
+pub enum Operations {
+    /// Gets localpv-hostpath resources.
+    #[clap(subcommand)]
+    Get(HosthpathGet)
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(hostpathget) => {
+                hostpathget.execute(cli_args).await?;
+            }
+        }
+        Ok(())
+    }
+}
+
+#[derive(clap::Subcommand, Debug)]
+pub enum HosthpathGet {
```

Comment:
Why not just Get? Since these are individual modules.

Reply:
Its being called by `enum Operations { Get(HostpathGet) }` . Cant have enum variant refer another enum using same name. Part of same file also.